README.md

```diff
@@ -457,31 +339,65 @@ This section illustrates the protocol flow for the case of communication using f
   Host: server.example.com
 ```
 
-#### Claims parameter
+#### Claims parameter (vp_jwt)
+
+Below is a non-normative example of how the `claims` parameter can be used for requesting verified presentations signed as JWT.
 
 ```json
 {
-    "vp_token": {
-      "format": "json-ld",
-      "claims":
-      {
-        "given_name": null,
-        "family_name": null,
-        "birthdate": null
+   "userinfo":
+    {
+      "vp_jwt": {
+        "claims":
+        {
+          "given_name": null,
+          "family_name": null,
+          "birthdate": null
+        }
       }
+    },
+   "id_token":
+    {
+     "auth_time": {"essential": true},
     }
 }
 ```
 
-### Authentication Response
+#### Claims parameter (vp_ldp)
```

Comment:
I suggest to show associated claims parameter example and token response example in a row.

vp_jwt
- claims 
- token response
- vp

vp_ldp
- claims
- token response
- vp

Reply:
vp is quite different ,but in claims, vp_jwt simply changes to vp_ldp and token response is the same

Reply:
restructured as above